client-js/app/elements/labrad-registry.ts

```diff
@@ -270,7 +292,8 @@ export class LabradRegistry extends polymer.Base {
     newKeyElem.value = '';
     newValueElem.value = '';
     dialog.open();
-    window.setTimeout(() => newKeyElem.$.input.focus(), 0);
+
+    this.bindIronAutogrowTextAreaResizeEvents(dialog, newValueElem);
```

Comment:
This is getting called every time the dialog opens, which means we add a new 'iron-overlay-closed' listener to the dialog every time the dialog is opened. I would suggest instead doing initialization work in the attached callback of the registry element itself; you can add a listener to each textarea that looks for the value changes and fires a resize event, and there's no need to remove this listener since it will just go away when the textarea element is destroyed.